python/mlc_llm/loader/huggingface_loader.py

```diff
@@ -115,13 +115,15 @@ def load(
         mlc_names = _loading_order(self.extern_param_map, self.torch_to_path)
         for mlc_name in tqdm(mlc_names):
             param = self._load_mlc_param(mlc_name, device=device)
-            if preshard_funcs is not None and mlc_name in preshard_funcs:
```

Comment:
keep this as well. The difference of this and the upstream is that, previously we applied presharding before quantization, each shard is quantized separately, this is not the expected behavior and it's also inconsistent with the behavior presharding is turned off, the upstream has fixed this behavior, it applied quantization first, and then presharding. However, current quantization depends on such behavior. We still need to keep this behavior for now

Reply:
I see. Is someone working on this?